### PR TITLE
Autofill cleanup tasks

### PR DESCRIPTION
**Pull request by DavidBiddle**
#### What problem does the pull request solve?
Resolves some divergences from the design, namely:
On the answer type page:
- makes the hint text for each input match the design
- hides the name input behind the feature flag (this was wrongly made visible)
On the answer settings pages:
- Adds a ‘Go to your questions’ link at the bottom of the page
On the name settings page:
- Change title list to `Mr, Mrs, Miss, Ms, Mx`

Trello card: https://trello.com/c/nPbS36yK/483-autofill-cleanup-tasks

#### Checklist

- [x] I've used the pull request template
- [x] I've linked this PR to the relevant issue (if mission work)
- [n/a] I've written unit tests for these changes (if code change)
- [n/a] I've updated the documentation in (If any documentation requires updating)
  - [n/a] README.md
  - [n/a] Elsewhere (please link)

### CODE REVIEW

File: app/models/page.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Page < ActiveResource::Base
   self.include_format_in_path = false
   headers["X-API-Token"] = ENV["API_KEY"]
 
-  ANSWER_TYPES = %w[name single_line number address date email national_insurance_number phone_number long_text selection organisation_name text].freeze
+  ANSWER_TYPES = %w[name organisation_name email phone_number national_insurance_number address date selection number text single_line long_text].freeze
```
Comment (aliuk2012) [inline app/models/page.rb:7]: @DavidBiddle do you you think now might be a good time to do the refactor, we spoke about this morning? Splitting this out into simple/complex and then this constant would just be a merge of simple/complex so we have a complete set of answer types?

**Comment by aliuk2012** [comment]
Just a suggestion about refactoring the answer type constant but otherwise looks good

File: app/models/page.rb (hunk 1 above)
Comment (DavidBiddle) [reply app/models/page.rb:7]: @aliuk2012 I think I'd prefer to do that when we remove the feature flag, as I think there's some related work involved there. But I've renamed `COMPLEX_ANSWER_TYPES` to `ANSWER_TYPES_WITH_SETTINGS` to at least make that clearer here

Comment (aliuk2012) [reply app/models/page.rb:7]: Thank you thats much clearer

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**